.github/workflows/cleanup_issues_before_2021.yml

```diff
@@ -0,0 +1,60 @@
+name: Close Pre-2021 Issues
+on:
+  # schedule:
+    # - cron: '35 15 * * *'  # Runs daily at 3:30 PM - stop after first run
+  workflow_dispatch: # Allows manual triggering from the Github UI
+
+permissions:
+  issues: write  # Required to close issues and post comments
+  contents: read  # Needed by most GitHub Actions
+  pull-requests: read  # Needed to access PR data
+
+jobs:
+  close-inactive:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
```

Comment:
Shouldn't need to checkout the repo unless we are loading a script or something from the repo